schedule/kernel/agama_prepare_baremetal.yaml

```diff
@@ -0,0 +1,13 @@
+name: agama-prepare-baremetal
+description:  >
+    Universal baremetal installation using ipxe_install and baremetal support
+vars:
+    AGAMA: 1
+    AGAMA_PRODUCT_ID: SLES
+    INST_AUTO: yam/agama/auto/sles_default.jsonnet
```

Comment:
We should probably stop using variable definition in this place. It is better to define it at test suite level. It is more user friendly, that variables are visible in settings page. Otherwise we need to dig into vars.json.